webapp/src/main/webapp/iaf/gui/js/controllers.js

```diff
@@ -269,7 +269,7 @@ angular.module('iaf.beheerconsole')
 						adapter.status = "stopped";
 
 					//Add flow diagrams
-					adapter.flow = Misc.getServerPath() + 'iaf/api/adapters/' + adapter.name + "/flow";
+					adapter.flow = Misc.getServerPath() + 'iaf/api/adapters/' + adapter.name + "/flow?"+adapter.upSince;
```

Comment:
It would be better to use a version tag, so that flow diagram cache will survive a restart